refinery/ui/source/js/commons/services/assay-file.js

```diff
@@ -11,11 +11,12 @@ angular
         },
         {
           query: {
-            method: 'GET'
-          },
-          params: { // TODO: I believe params should be inside query?
-            limit: 'limit',
-            offset: 'offset'
+            method: 'GET',
+            params: {
+              limit: 'limit',
+              offset: 'offset',
+              data_set_uuid: ''
```

Comment:
I think I remember you telling me the reason behind having this as a placeholder before, but could I get a refresher?